### ISSUE TITLE
Descriptor loss during training.

### ISSUE THREAD

**Issue by junshengzhou** [rsy6318/CorrI2P#2]
Hi, 
Thanks again for sharing your great job! I processed the Kitti data myself and reproduced CorrI2P. I find the **descriptor loss** is hard to drop and is around **3.8** and I am not sure if it's normal. so could you share your pretrained model and training log in Kitti dataset for my reference on the correctness of my reproduction.

**Comment by rsy6318** [comment]
How did you process KITTI?

**Comment by rsy6318** [comment]
I just check my training log, and the descriptor loss is around 3 after 20 epoches. 
Besedes, we have uploaded my dataset through Onedrive.

**Comment by junshengzhou** [comment]
Hi, 
I use the scripts provided by DeepI2P to process the KITTI dataset. And adjust it according to your dataloader.

**Comment by junshengzhou** [comment]
The descriptor loss starts from around **4**, right?  

Thanks for providing your dataset! I will try it !

**Comment by rsy6318** [comment]
Yes, it is around 3.9 at the beginning during my experiment.

**Comment by rsy6318** [comment]
DeepI2P's scripts has some problems.  
After spliced some frames of point cloud, some moving objects such as walking human, cars would not overlap, which would decrease the registration, and we have explained this in our paper.
Every frame of point cloud is dense enough to use it directly, and after voxel down sample, it is uniformly distribution from 0 to 80m. So it is necessary to splice them !

**Comment by junshengzhou** [comment]
Okay~ I get it.  I guess you mean 'it is not necessary to splice them', right?

Thanks for clearing this up! I will check the details in your paper again! And I will use your provided data for the later experiments, that really helps me a lot!!!

**Comment by rsy6318** [comment]
Yes, it is not necessary to splice them !

**Comment by junshengzhou** [comment]
Hi, 
Thanks for your quick reply. Is the data `data = np.load(os.path.join(pc_folder, '%06d.npy' % seq_i))` the same as DeepI2P except the accumulation operation?  I did't find the cam-nwu transform in your dataloader, maybe I should involve the transform when I use the data of DeepI2P?

**Comment by rsy6318** [comment]
Thanks for remind. I have upload camera calib.
